_docs/concepts/what-is-istio/goals.md

```diff
@@ -8,7 +8,9 @@ layout: docs
 type: markdown
 ---
 
-This page the core principles that guide Istio's design.
+This page outlines the core principles that guide Istio's design and explains some key design decisions based on these goals.
+
+## Design goals
```

Comment:
Yeah, I think I was thinking of adding another section and then decided not to. Done.